src/System.Threading.Channels/ref/System.Threading.Channels.netcoreapp.cs

```diff
@@ -13,6 +13,6 @@ public partial class ChannelClosedException : System.InvalidOperationException
     }
     public abstract partial class ChannelReader<T>
     {
-        public virtual System.Collections.Generic.IAsyncEnumerable<T> ReadAllAsync() { throw null; }
+        public virtual System.Collections.Generic.IAsyncEnumerable<T> ReadAllAsync([System.Runtime.CompilerServices.EnumeratorCancellationAttribute] System.Threading.CancellationToken cancellationToken = default) { throw null; }
```

Comment:
nit: It seems ok to put the attribute there, as a form of documentation, but it won't do anything. The attribute must be placed on implementations to generate the desired `GetAsyncEnumerator` code.